docs/source/getstart.rst

```diff
@@ -0,0 +1,227 @@
+    * if use :ref:`chainerui's LogReport <module_log_report>`, only ``elapsed_time`` is set.
+* The other key-value items are plotted.
+
+Setup example, a brief extract taken from `MNIST example <https://github.com/chainer/chainerui/blob/master/examples/train_mnist.py>`__:
+
+.. code-block:: python
+
+  import chainer.links as L
+  from chainer import training
+  from chainer.training import extensions
+
+  def main():
+      # Classifier reports softmax cross entropy loss and accuracy at every
+      # iteration
+      # [ChainerUI] plot loss and accuracy reported by this link
+      model = L.Classifier(MLP(args.unit, 10))
+
+      trainer = training.Trainer(updater, (args.epoch, 'epoch'), out=args.out)
+
+      # [ChainerUI] read 'log' file for plotting values
+      trainer.extend(extensions.LogReport())
+
+created ``log`` file example::
+
+  [
+      {
+          "main/loss": 0.1933198869228363,
+          "validation/main/loss": 0.09147150814533234,
+          "iteration": 600,
+          "elapsed_time": 16.052587032318115,
+          "epoch": 1,
+          "main/accuracy": 0.9421835541725159,
+          "validation/main/accuracy": 0.9703000783920288
+      }, 
+      {
+          "main/loss": 0.07222291827201843,
+          "validation/main/loss": 0.08141259849071503,
+          "iteration": 1200,
+          "elapsed_time": 19.54666304588318,
+          "epoch": 2,
+          "main/accuracy": 0.9771820902824402,
+          "validation/main/accuracy": 0.975399911403656
+      },
+      ...
+  ]
+
+without ``Trainer`` code example, a  brief extract take from `MNIST custom loop example <https://github.com/chainer/chainerui/blob/master/examples/train_mnist_custom_loop.py>`__:
+
+.. code-block:: python
+
+  from chainerui.utils import LogReport
+
+  def main():
+
+      # [ChainerUI] setup log reporter to show on ChainerUI along with 'args'
+      ui_report = LogReport(args.out, conditions=args)
+      while train_iter.epoch < args.epoch:
+
+          # ...train calculation
+
+          if train_iter.is_new_epoch:
+
+              # [ChainerUI] write values to 'log' file
+              stats = {
+                  'epoch': train_iter.epoch,
+                  'iteration': train_iter.epoch * args.batchsize,
+                  'train/loss': train_loss, 'train/accuracy': train_accuracy,
+                  'test/loss': test_loss, 'test/accuracy': test_accuracy
+                  }
+              ui_report(stats)
+
+.. note::
+
+   :ref:`chainerui's LogReport <module_log_report>` only sets ``exampled_time``, so train loop has to set ``epoch`` and ``iteration`` manually.
+
+Experimental conditions
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: ../images/result_table_condition_sample.png
+
+ChainerUI shows training job with experimental conditions read from ``args`` file. ``args`` file is a JSON file, which includes key-value pairs. See :ref:`save_args <module_save_args>`, util function to dump command line arguments or dictionary to ``args`` file.
```

Comment:
...shows the training job with experimental conditions read from the ... or dictionaries...